flake.nix

```diff
@@ -471,6 +471,23 @@
         }
       );
 
+      apps = forAllSystems (
+        system:
+        let
+          pkgs = nixpkgsFor.${system}.native;
+          opener = if pkgs.stdenv.isDarwin then "open" else "xdg-open";
```

Comment:
Any reason not to `lib.getExe' xdg-utils "xdg-open"` here so that even if it's not installed it'll still work?

(I'm fine with depending on `open` already being in PATH for macOS since, well, yeah)